packages/jaeger-ui/src/components/TracePage/TraceStatistics/index.tsx

```diff
@@ -479,12 +355,26 @@ export default class TraceStatistics extends Component<Props, State> {
         {this.state.showPopup ? (
           <PopupSQL closePopup={this.togglePopup} popupContent={this.state.popupContent} />
         ) : null}
-        <table className="test1893">
-          <tbody className="DetailTraceTableTbody--TraceStatistics">
-            {this.renderTableHead()}
-            {this.renderTableData()}
-          </tbody>
-        </table>
+        <Table
+          className="span-table span-view-table"
+          columns={columns}
+          dataSource={withoutDetail}
+          pagination={{
+            total: withoutDetail.length,
+            pageSizeOptions: ['10', '20', '50', '100'],
+            showSizeChanger: true,
+            showQuickJumper: true,
+          }}
+          rowClassName={row => {
+            if (row.type === 'undefined') {
```

Comment:
What is the meaning of row.type? What are the possible values?

Reply:
When you subgroup the rows based on (say http.status code here)in that subgroup if there are no spans with that tag then those rows are of type undefined.
let us see in this pic now the row under the mysql row is of type undefined and row under the customer row is of type defined
And possible values are defined and undefined
![new1](https://github.com/jaegertracing/jaeger-ui/assets/116095646/1cda7c20-4b2e-4128-9c0e-4efb4cabd79e)

Reply:
Thanks, now it makes sense. Sounds like `type` is a terrible name to describe this concept -- what you have instead is a Boolean indicator telling whether the row has a subgroup label defined or not (the catch-all case), e.g. `hasSubgroupValue`. Can we change this and make a corresponding comment in the type declaration?

Reply:
Yes that might be a good idea , I will make those changes in code